libstuff/libstuff.h

```diff
@@ -242,28 +239,23 @@ inline void SLogLevel(int level) {
     _g_SLogMask = LOG_UPTO(level);
     setlogmask(_g_SLogMask);
 }
-inline bool SLogLevelIsSet(int level) { return (_g_SLogMask & (1 << level)); }
```

Comment:
Even if we're not using it now, this is a helpful function at times when debugging.  For example, sometimes there is expensive logic to be performed to construct a good log line, so you only want to run it if you know the resulting line will be logged.  I could believe we don't use it now, but I do know there are scenarios in the past where I've used it.